### PR TITLE
pacman: use short options only

### PR DESCRIPTION
**Pull request by marchersimon**
For #8046 
I felt like the whole issue is mostly about `pacman`.
So we should start with this one and see if there still are other commands where it is necessary to include short options.
I put the short options in token syntax for now, but we can debate about that.

### CODE REVIEW

**Comment by CleanMachine1** [comment]
Personally don't like [this current version](https://github.com/tldr-pages/tldr/pull/8286/commits/bbd431c01629241305785d44ed9ff7f4a636cf77).

I think we just pick one, and since its such a common command, maybe we should go with `sudo pacman -Syu`. I've probably said something else in the past as well.

**Comment by marchersimon** [comment]
I just think that it's useful to have the long version, since you directly know what all options do and what would happen if you left out a specific one. But I totally get your point, that it looks weird.

**Comment by kbdharun** [comment]
> I think we just pick one, and since it's such a common command, maybe we should go with `sudo pacman -Syu`. I've probably said something else in the past as well.

Yeah, I think the same too, Even though long options are useful and very informative. I think many would prefer shorter options as it is the one that is widely used. I think the [current implementation](https://github.com/tldr-pages/tldr/pull/8286/commits/bbd431c01629241305785d44ed9ff7f4a636cf77) might confuse users about which command to execute. I went through the issue about it fully and I noticed there were a lot of suggestions and proposed solutions. 

I agree with @CleanMachine1 as this is a special case it is better to use shorter options on the page if users need to refer to longer options they can use the man link provided with `More information:`.

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks @marchersimon

**Comment by EmilySeville7cfg** [comment]
I don't like idea to use short options when there are long options supported by command. From my perspective it's better to copy `pacman` page to `common/` and use short options there but not here. Either way it depends on what can be placed in `common/`. And I am going to discuss it in out gitter chat. I'll leave link here for my message.

**Review by EmilySeville7cfg — CHANGES_REQUESTED** [review]
I don't like idea to use short options when there are long options supported by command. From my perspective it's better to copy `pacman` page to `common/` and use short options there but not here. Either way it depends on what can be placed in `common/`. And I am going to [discuss](https://gitter.im/tldr-pages/tldr?at=62f0f457aa09177429231d33) it in out gitter chat.

**Comment by marchersimon** [comment]
I don't see why we should put this into `common`, it's only available on `linux`. Also, if you are on linux and look up the tldr page for `pacman`, you will only see the `linux` version, not the `common` version.

**Comment by kbdharun** [comment]
> I don't see why we should put this into `common`, it's only available on `linux`. Also, if you are on linux and look up the tldr page for `pacman`, you will only see the `linux` version, not the `common` version.

+1, I too think the same.

**Review by sbrl — APPROVED** [review]
The `pacman` page should *not* be in `common`. `common` is only for pages about commands that are supported by **more than 1 platform**.

Further, as I have explained in another issue recently, we tldr pages should document usages of commands that are *useful*. Sure, it's helpful to know what the short options stand for, but there are other ways to achieve that than requiring long options everywhere. `sudo pacman --sync --refresh --sysupgrade` is an enormous mouthful to remember and type every time - I'll bet most users use `sudo pacman -Syu` instead.

We never used to require long options on all pages, and that particular guideline was never intended to mean "use long options everywhere". I can dig up multiple references to this if there's interest.

**Comment by EmilySeville7cfg** [comment]
> The pacman page should not be in common. common is only for pages about commands that are supported by more than 1 platform.

@sbrl, what about not to use common at all? It's not clear what platforms on some command is available when it is in `common/`?

> Further, as I have explained in another issue recently, we tldr pages should document usages of commands that are useful. Sure, it's helpful to know what the short options stand for, but there are other ways to achieve that than requiring long options everywhere. sudo pacman --sync --refresh --sysupgrade is an enormous mouthful to remember and type every time - I'll bet most users use sudo pacman -Syu instead.

Let's define subset of pages where we prefer short options. For instance in package manager pages we can prefer short options anywhere. I can't decide whether to approve or decline such PRs without formal rules. I have nothing to refer to. Note that when I block merging by requesting changes it doesn't mean that I don't like changes made in PR at all. I just wanna everything to be standardized.

**Comment by kbdharun** [comment]
> Let's define subset of pages where we prefer short options. For instance in package manager pages we can prefer short options anywhere. I can't decide whether to approve or decline such PRs without formal rules. I have nothing to refer to. Note that when I block merging by requesting changes it doesn't mean that I don't like changes made in PR at all. I just wanna everything to be standardized and that's why I am here.

Yeah, it would be better to specify the use cases for short commands over longer ones for PR's, maybe we can document the edge cases too in `contributing.md`. What is your opinion? @marchersimon @CleanMachine1

File: pages/linux/pacman.md [hunk 1]
```diff
@@ -6,32 +6,32 @@
 
 - Synchronize and update all packages:
 
-`sudo pacman --sync --refresh --sysupgrade`
+`sudo pacman -Syu`
```
Comment (EmilySeville7cfg) [inline pages/linux/pacman.md:9]: All short options should be described via square brackets where possible here and in all translations.

**Review by EmilySeville7cfg — COMMENTED** [review]
Please describe all short options via square brackets in all pages changed in PR.

**Review by navarroaxel — APPROVED** [review]

**Comment by EmilySeville7cfg** [comment]
> Yeah, it would be better to specify the use cases for short commands over longer ones for PR's, maybe we can document the edge cases too in contributing.md. What is your opinion? @marchersimon @CleanMachine1

Style guide maybe the better place to document it.

**Review by EmilySeville7cfg — APPROVED** [review]

**Comment by kbdharun** [comment]
> Style guide maybe the better place to document it.

Yeah, That would be good. To place it under special cases under style guide.

**Pull request merged**